src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoTreeNodesDefinition.xml

```diff
@@ -64,16 +64,49 @@
   
   <Node Name="Database" LocLabel="string.Empty" Image="Database" BaseClass="ModelBased" NodeType="Database" IsAsyncLoad="" Strategy="CreateModel" TreeNode="ExpandableSchemaTreeNode" PutFoldersAfterNodes="true">
     <ConditionalChildQuerierType Name="SqlSchema" SettingsFlag="GroupBySchema"/>
+    <Filters>
+      <!-- 
+        Excluding built-in schemas for backward compatibility from the database node and moving them to legacy schema folder.
+        The list of schemas was obtained from: 
+        https://learn.microsoft.com/en-us/sql/relational-databases/security/authentication-access/ownership-and-user-schema-separation?view=sql-server-ver16#built-in-schemas-for-backward-compatibility
+      -->
+      <Filter Property="Name" Value="db_accessadmin" Type="string" FilterType="contains" IsNotFilter="true"/>
```

Comment:
You shouldn't be using contains IMO - what if someone had their own schema `db_accessadmin2`? Unlikely, sure. But is there a reason to go with contains over a strict equals?

Reply:
Oh yeah, good catch. At first, I was doing name contains "db_" and had thought of a similar scenario.